Lib/timeit.py

```diff
@@ -77,9 +77,11 @@ def inner(_it, _timer{init}):
     return _t1 - _t0
 """
 
+
 def reindent(src, indent):
     """Helper to reindent a multi-line statement."""
-    return src.replace("\n", "\n" + " "*indent)
+    return src.replace("\n", "\n" + " " * indent)
```

Comment:
Revert.  See PEP 8, Other Recommendations
```suggestion
    return src.replace("\n", "\n" + " "*indent)
```

Reply:
Pycharm default formatter add spaces ignore the priority of statements.Any good advice?